pkg/cluster/cluster.go

```diff
@@ -126,6 +126,11 @@ type AddOnItem struct {
 	Available bool
 }
 
+type lmtSprReasonItem struct {
+	ID      string
+	Summary string
+}
```

Comment:
Can we also include the `details` field as well?

Reply:
Not for now. maybe in another card for this, because then we have to mess with the table generation code or find a solution to output it differently (example create a details flag in order to change from a table format to list or find a way to make the table not wrap around). Issue is the table details column is long and overlaps a lot and makes the whole thing cluttered. 

Now, if we don't care about how pretty it looks i have no problem, we can add it. but then the UX for this command would be terrible.